src/cli.ts

```diff
@@ -188,7 +190,7 @@ export async function cli(
           )
         }
         if (mode !== 'valid' && opt.json) {
-          console.log(JSON.stringify(opt.single ? data[0] : data))
+          console.log(JSON.stringify(opt.single ? data[0] : data, null, Number(opt.indent)))
```

Comment:
Note also the other places where output is being generated: lines 97, 115, and 164 will need to be updated. It'd probably be best to determine a single `jsonIndent` value somewhere above, and use that for the `JSON.stringify()` calls, and handle the yaml Document's `toString{ indent })` value separately.